app/components/Send/SendPanel/index.jsx

```diff
@@ -35,7 +35,7 @@ type Props = {
   handleSubmit: () => any,
   handleSend: () => any,
   clearErrors: (index: number, field: string) => any,
-  addRow: (row: Object) => any,
+  addRow: (row?: Object) => any,
```

Comment:
When I remove this change on your branch I do not see any flow errors I think we should leave this as is as row must always be defined (and is always defined)